### DISCUSSION TITLE
[V5] Custom Select missing properties - Typescript upgrade

### DISCUSSION THREAD

**Discussion by pmartiner** [JedWatson/react-select#4848] [Q&A] [answered]
Hi, everyone!
First of all, let me congratulate you for the amazing Typescript rewrite! It's been much much easier to work with those types compared with de DT ones. 
Currently we have a project that has a custom `Select` component, but, since upgrading to v5.1.0, every time we try to use it, TS throws a `is missing the following properties` error,  but I don't know why. Everything looks good on the Select component. I was wondering if anyone could help us figure this one out. Thanks!

Here's my Codesandbox: https://codesandbox.io/s/selectcontrol-error-74sm1
The error can be seen on the `App.tsx` file

**Comment by Methuselah96** [comment, marked as answer]
Thanks, we put a lot of work into it and love to get feedback!

In your `components/SelectControl/index.ts` try importing `Props` from `react-select` instead of the declaration file. That seems to [fix it for me](https://codesandbox.io/s/selectcontrol-error-forked-e1uld?file=/src/components/SelectControl/index.tsx).

**Reply by pmartiner** [reply]
Thank you very much, @Methuselah96! That solved it!
